File: DownstreamService/src/main/java/com/stackroute/DownstreamService/domain/Chicklets.java
package com.stackroute.DownstreamService.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Chicklets {
    private Qualification qualification;
    private Institution institution;
    private String summary;
    private Skills skill;
    private CurrentLocation currentLocation;
    private PastLocation pastLocation;
}

File: angular/src/app/cards/locationclasses/chicklets.ts
import { CurrentLocation } from './currentlocation';
import { PastLocation } from './pastlocation';

export class LocationChicklets {
    constructor(private currentLocation: CurrentLocation,
        private pastLocation: PastLocation[]) {
        this.currentLocation = currentLocation;
        this.pastLocation = pastLocation;
    }
}
